docs/resources/application_profile.md

```diff
@@ -1,33 +1,149 @@
+  version = "1.18.14"
+}
+
+data "spectrocloud_pack" "ubuntu" {
+  name = "ubuntu-gcp"
+  version  = "1.0.0"
+}
+
+resource "spectrocloud_cluster_profile" "profile" {
+  name        = "gcp-picard-2"
+  description = "basic cp"
+  cloud       = "gcp"
+  type        = "cluster"
+
+  pack {
+    name   = "spectro-byo-manifest"
+    tag    = "1.0.x"
+    uid    = data.spectrocloud_pack.byom.id
+    values = <<-EOT
+      manifests:
+        byo-manifest:
+          contents: |
+            # Add manifests here
+            apiVersion: v1
+            kind: Namespace
+            metadata:
+              labels:
+                app: wordpress
+                app3: wordpress3
+              name: wordpress
+    EOT
+  }
+
+  pack {
+    name   = "csi-gcp"
+    tag    = "1.0.x"
+    uid    = data.spectrocloud_pack.csi.id
+    values = data.spectrocloud_pack.csi.values
+  }
+
+  pack {
+    name   = "csi-gcp"
+    tag    = "1.0.x"
+    uid    = data.spectrocloud_pack.csi.id
+    values = data.spectrocloud_pack.csi.values
+  }
+
+  pack {
+    name   = "cni-calico"
+    tag    = "3.16.x"
+    uid    = data.spectrocloud_pack.cni.id
+    values = data.spectrocloud_pack.cni.values
+  }
+
+  pack {
+    name   = "kubernetes"
+    tag    = "1.18.x"
+    uid    = data.spectrocloud_pack.k8s.id
+    values = data.spectrocloud_pack.k8s.values
+  }
+
+  pack {
+    name   = "ubuntu-gcp"
+    tag    = "LTS__18.4.x"
+    uid    = data.spectrocloud_pack.ubuntu.id
+   
+```
+
+
+###  App Profile with Helm Chart
+
+```hcl
+data "spectrocloud_registry_helm" "helm" {
+  name = "Public Spectro Helm Repo"
+}
+
+data "spectrocloud_pack" "marvel-app" {
+  name         = "marvel-app"
+  registry_uid = data.spectrocloud_registry_helm.helm.id
+  version      = "0.1.0"
+}
+
+resource "spectrocloud_application_profile" "my-app-profile" {
+  name        = "my-app-profile"
+  description = "A profile for a simple application"
+  context     = "project"
+  pack {
+    name            = data.spectrocloud_pack.marvel-app.name
+    type            = "helm"
+    registry_uid    = data.spectrocloud_registry_helm.helm.id
+    source_app_tier = data.spectrocloud_pack.marvel-app.id
+  }
+  tags = ["name:marvel-app", "terraform_managed:true", "env:dev"]
+}
+```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `name` (String)
-- `pack` (Block List, Min: 1) (see [below for nested schema](#nestedblock--pack))
+- `name` (String) Name of the application profile
+- `pack` (Block List, Min: 1) A list of packs to be applied to the application profile. (see [below for nested schema](#nestedblock--pack))
 
 ### Optional
 
-- `cloud` (String)
-- `context` (String)
-- `description` (String)
-- `tags` (Set of String)
+- `cloud` (String) The cloud provider the app profile is eligble for usage. Default value is `all`.
+- `context` (String) Context of the application profile. Allowed values are `project`, `cluster`, or `namespace`. Default value is `project`.
```

Comment:
allowed values are project cluster and system. namespace is not allowed.

Reply:
@nikchern fixed, nice catch 😄